components/TabbedView/TabbedView.jsx

```diff
@@ -120,7 +193,9 @@ class TabbedView extends React.Component {
               aria-controls={`${this.sanitize(title)}-panel`}
               aria-selected={title === activeTab}
             >
-              {title}
+              {icon ? <IconContainer>{icon}</IconContainer> : null}
```

Comment:
if we have only one option, how about of use 

```
{icon && <IconContainer>{icon}</IconContainer>}
```
instead ternary?

PS: same to below

Reply:
Updated